lib/media/vtt_text_parser.js

```diff
@@ -39,6 +39,12 @@ shaka.media.VttTextParser =
     function(data, offset, segmentStartTime,
              segmentEndTime, useRelativeCueTimestamps) {
 
+  if (segmentStartTime > 0) {
```

Comment:
I was unsure about this and wanted feedback. I thought it might be desirable for all users to get this warning since deprecating this will affect users whether they are using the default setting or not. But if that seems like overkill than I can put that second condition back in.